.github/workflows/main.yml

```diff
@@ -18,7 +18,9 @@ jobs:
     - name: Install
       run: npm install
     - name: Lint
-      run: npm run lint -- --no-fix
+      run: |
+        npm run lint -- --no-fix
+        DEBUG="lockfile-lint,validate-host-manager" npx lockfile-lint --path package-lock.json --validate-https --allowed-hosts npm
```

Comment:
This `DEBUG=$function/$logname/etc` syntax is from the Node [`debug`](https://www.npmjs.com/package/debug) package :+1: normally people just `DEBUG=*`, but shouldn't matter much as we know which functions should appear for the linter.